components/common/Flex.tsx

```diff
@@ -8,7 +8,8 @@ interface FlexProps {
   align?: 'start' | 'end' | 'center' | 'flex-end' | 'flex-start' | 'self-start' | 'self-end' | 'stretch';
   justify?: 'start' | 'end' | 'center' | 'space-between' | 'space-around' | 'space-evenly' | 'left' | 'right'
   direction?: 'row' | 'column' | 'row-reverse' | 'column-reverse'
-  gap?: number | string
+  gap?: number | string,
+  wrap?: 'nowrap' | 'wrap' | 'wrap-reverse'
```

Comment:
Out of curiosity, could it be better to replace these with types from `csstype` library? The library seems to be already included in our repo and I was able to build this

```suggestion
import {
  AlignItemsProperty,
  FlexDirectionProperty,
  GridGapProperty,
  JustifyItemsProperty
} from 'csstype'

interface FlexProps {
  align?: AlignItemsProperty
  justify?: JustifyItemsProperty
  direction?: FlexDirectionProperty
  gap?: GridGapProperty<number>
}
```

Reply:
Good call @sangwoo108 ! Looks like it's in `package.json`, so I should definitely be using it!